### ISSUE TITLE
ConfigOverrides are not being applied in Azure Cluster

### ISSUE THREAD

**Issue by ivanhernandez** [Azure/service-fabric-issues#724]
A stateless, singleton service being deployed into Azure with ConfigOverrides, is taking default values declared in application manifest parameters list. Values in application parameters file are not being take into consideration.

When deploying the same application into local 1 node cluster, using the same application parameters file (local1node publish profile makes use of the same parameters file), it works ok (application instance running at local cluster reads overriden values instead of default ones).

 * Azure cluster is at version 6.1.445.9494. It is a 2016 datacenter. _The application works on .Net Fw 4.6.2_
 * Local cluster is at version 6.0.232.9494.
* Application is including Microsoft.ServiceFabric nuget 6.0.232 _(upgrading to 6.1.232-preview does not fix the problem)_
* Microsoft Azure Service Fabric SDK locally installed is 2.8.232

**Comment by ivanhernandez** [comment]
Hi, @RajeetN  Is there any progress on this issue?

**Comment by RajeetN** [comment]
Sorry for the late response. Does your cloud.xml have the right parameters set?

**Comment by ivanhernandez** [comment]
Yes, it does. It is a copy of the local config file, with some different
values here and there. When I copy and paste the whole local config file
into cloud.xml, behaviour is as described in the issue: local node runs the
config pasted from cloud.xml, but when deploying at Azure, process takes
DefaultValues from manifest instead of rhose at cloud.xml.

There is a new SF version in Azure (6.1.4xx, I believe) so I am going to
try upgrading the cluster to see if this was fixed for that version.

**Comment by RajeetN** [comment]
@dbreshears , could you take a look? Thanks!

**Comment by dbreshears** [comment]
@ivanhernandez, In your PublishProfiles\Cloud.xml file, what is specified in the ApplicationParameterfile element?  Is should be pointing to Cloud.xml however may be set to something else?     If that isn't the issue, can you zip up and share all the XML files in your solution,  manifests, settings, publish profiles, parameters,etc..

**Comment by ivanhernandez** [comment]
Of course, each publish file uses its own config.

Can't send you those configs as I don't actually have access to them and
they contain some private data. Instead, I will try to generate a minimal
project reproducing the case, for your study on monday.

**Comment by dbreshears** [comment]
That would be great.  Will see if I can repro as well.

**Comment by dbreshears** [comment]
@ivanhernandez,  I wasn't able to reproduce this.  Are you still having issues or did you get them resolved?

**Comment by carl-ashby-silvertech** [comment]
I'm experiencing the same issue, was this ever resolved?

**Comment by haseeb-ahmed-tkxel** [comment]
I recently stated having this issue. For me, deleting the SF app and recreating it fixes it rarely. So I have to keep trying until it works which is quite annoying BTW.
